assets/js/components/starRating.js

```diff
@@ -0,0 +1,22 @@
+class StarRating {
+    constructor(target, stars = 5) {
```

Comment:
Why default stars are 5?

Reply:
For now stars are used only in the rating system, e.g. 5 stars.